test/dotnet-msbuild.Tests/GivenDotnetMSBuildBuildsProjects.cs

```diff
@@ -77,6 +86,37 @@ public void When_help_is_invoked_Then_MSBuild_extra_options_text_is_included_in_
         }
 
         [Fact]
+        public void WhenRestoreSourcesStartsWithUnixPathThenHttpsSourceIsParsedCorrectly()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
We have this guy https://github.com/dotnet/cli/blob/rel/1.0.0/test/Microsoft.DotNet.Tools.Tests.Utilities/WindowsOnlyFactAttribute.cs for Windows.